README.md

```diff
@@ -793,9 +793,9 @@ For example, the following will auto-download the [EssentialsX](https://www.spig
                               |
                               +-- project slug
   ```
-  Also, specific version can be declared using colon symbol and version id after the project slug. For instance:
+  Also, specific version/type can be declared using colon symbol and version id/type after the project slug. The version id can be found at 'Metadata' section. Valid version types are ``RELEASE``, ``BETA`` ``ALPHA``. For instance:
```

Comment:
I can't remember if that specific enum is processed case insensitive. In any case, good to be consistent with lowercase. 

```suggestion
  Also, specific version/type can be declared using colon symbol and version id/type after the project slug. The version id can be found at 'Metadata' section. Valid version types are `release`, `beta`, `alpha`. For instance:
```